src/proxy.cpp

```diff
@@ -46,8 +46,9 @@
 #include "socket_base.hpp"
 #include "err.hpp"
 
+// TODO: determine if this is an issue, since zmq.h is being loaded from pch.
 // zmq.h must be included *after* poll.h for AIX to build properly
```

Comment:
I wanted to highlight this comment. The previous implementation did not conform to the requirement indicated by the comment. I don't have the ability to test this, but this pull request does not change the actual behavior.